tools/clang/test/HLSLFileCheck/dxil/intrinsics/launch_types_broadcast.ll

```diff
@@ -0,0 +1,50 @@
+; RUN: %dxv %s 
+
+; There should be no errors!
```

Comment:
If we have a succeeding case for broadcasting, it seems we should have ones for coalescing and thread launch as well.

However, this could be considered covered by tests specifically for these intrinsics in those launch modes that should exist elsewhere, rather needing validation tests which don't fail validation here.

Reply:
I'm not sure what the resolution to this comment was.  I don't think this test should be here.  If anything, just make sure we have tests elsewhere that check the appropriate system value intrinsics are called when the system values are used (not as a validation test).

Reply:
@bob80905, it looks like @tex3d had further comments here. How has this been addressed?

Reply:
I resolved this comment by removing the broadcast test from the PR. I agreed with Tex; it doesn't need to be tested, because it already works and is tested by the other tests we have. This PR doesn't add any extra diagnostics in the broadcast launch type case.